man/ocitools-generate.1.md

```diff
@@ -84,6 +84,45 @@ inside of the container.
   The special *PATH* `host` removes any existing IPC namespace from the
   configuration.
 
+**--linux-cpu-shares**=CPUSHARES
+  Specifies a relative share of CPU time available to the tasks in a cgroup.
+
+**--linux-cpu-period**=CPUPERIOD
+  Specifies a period of time in microseconds for how regularly a cgroup's access to CPU resources should be reallocated (CFS scheduler only).
```

Comment:
On Fri, Aug 05, 2016 at 11:58:48AM -0700, hmeng-19 wrote:
> @wking , I think there are two ways we can use to determine the current scheduler.
> 1) Read the kernel config file, refer to https://en.wikipedia.org/wiki/Completely_Fair_Scheduler:
> ```
> [hmeng@localhost boot]$ grep CONFIG_SCHED_AUTOGROUP /boot/config-4.6.4-201.fc23.x86_64 
> CONFIG_SCHED_AUTOGROUP=y
> ```
> 2) check the /sys/fs/cgroup/cpu dir to see whether `cpu.cfs_period_us` exists or not.

These both sound like “does your kernel support that scheduler”, but
if scheduler policy is per-thread, then how do you know the container
will *use* that available scheduler (and not another available
scheduler)?